### PR TITLE
NetAcuity Tool Uses NetAcuity Cloud API

### PR DESCRIPTION
**Pull request by hotline-emu**
#### Convert the NetAcuity Tool to utilize the NetAcuity Cloud API instead of sockets, per vendor's request.

JIRA: https://jira.traderinteractive.com/browse/EXODUS-252

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -14,7 +14,8 @@
         "php": "~5.6 || ~7.0",
```
Comment (chrisryan) [inline composer.json]: You have used some syntax that is not 5.6 compatible. We should probably remove the 5.6 dependency unless we want to change the code to support php 5.6. I would prefer we remove 5.6.

File: tests/NetAcuityTest.php [hunk 2]
```diff
@@ -11,58 +15,130 @@
...
+                'edge-postal-conf' => '99',
+                'edge-gmt-offset' => '-500',
+                'edge-in-dst' => 'n',
+                'edge-timezone-name' => 't3',
+            ]
+        );
+        $mockClient = $this->getMockGuzzleClient();
+        $mockClient->method('send')->willReturn($mockResponse);
+        $client = new NetAcuity($mockClient, 'a token', NetAcuity::NETACUITY_EDGE_DB_ID);
+        $actual = $client->getGeo('192.1681.1');
```
Comment (chrisryan) [inline tests/NetAcuityTest.php]: I'm sure this is a bogus IP but it's still incorrectly formatted.

Comment (hotline-emu) [reply tests/NetAcuityTest.php]: This is good to note, I will make it look appropriate.

File: composer.json (hunk 1 above)
Comment (hotline-emu) [reply composer.json]: Great catch

**Comment by hotline-emu** [comment]
I should have noted that I am currently abstracting most of this to extend it into the other 10+ databases they offer. They have been pushing us to use something called pulse as well. I'd hold off on reviewing further until I get that up here.

**Comment by hotline-emu** [comment]
@bayemane @chrisryan - This is ready for review.

**Pull request merged**